module-system/sov-modules-stf-template/src/app_template.rs

```diff
@@ -234,7 +238,7 @@ where
     }
 
     // Do all stateless checks and data formatting, that can be results in sequencer slashing
-    fn pre_process_batch(
+    pub(crate) fn pre_process_batch(
```

Comment:
Why is it became public for crate? I don't see places where it is used now, probably I am missing something, but want to double check